packages/frontend/src/components/chart/configure/effects/handleEffect.ts

```diff
@@ -80,15 +80,15 @@ function handleFetchAlternativeTvl(
 }
 
 function handleFetchTokenTvl(
-  { url, requestId, token }: FetchTokenTvlEffect,
+  { url, requestId, token, assetType }: FetchTokenTvlEffect,
   dispatch: (message: Message) => void,
 ) {
   timeoutLoader(requestId, dispatch)
   fetchThenDispatch(
     url,
     dispatch,
     // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
-    (data) => ({ type: 'TokenTvlLoaded', requestId, token, data }),
+    (data) => ({ type: 'TokenTvlLoaded', requestId, token, assetType, data }),
```

Comment:
wouldn't it break the production token fetching logic? because the detailed tvl is disabled on prod server

Reply:
Not really, we just assume that on prod every single token is EBV and off we go :+]